doc/python/facet-plots.md

```diff
@@ -31,8 +31,8 @@ jupyter:
     page_type: u-guide
     permalink: python/facet-plots/
     redirect_from:
-      - python/trellis-plots/
-      - python/facet-trellis/
+    - python/trellis-plots/
+    - python/facet-trellis/
```

Comment:
did you change this yourself or was it Jupyter? From a quick `grep` I think we have the two syntaxes in the doc directory, so this should be fine, but just checking.

Reply:
It was Jupyter, not me that edited the front matter.

Reply:
ok, :dancer:!